validator/validator-main.protoascii

```diff
@@ -2634,8 +2634,17 @@ tags: {
 }
 tags: {
   html_format: AMP
-  html_format: AMP4ADS
```

Comment:
removed a4a and ads for existing `kind=subtitle` rule of IMA since IMA itself is only `html_format: AMP` anyway.

Reply:
@honeybadgerdontcare Is this safe to do? I'm worried about invalidating amp4ads/actions pages that may have this in their HTML (even if it's a suboptimal runtime experience).

Reply:
Oh, nevermind, you mean https://github.com/ampproject/amphtml/blob/bccd9a769b3c20a95ddbf66019e4bd0f8d24dda6/extensions/amp-ima-video/validator-amp-ima-video.protoascii#L28. OK, sounds good.

Reply:
lgtm, thank you for being extra careful